src/stores/transactionDetails.js

```diff
@@ -69,11 +67,16 @@ export const useTransactionDetailsStore = defineStore('transactionDetails', () =
     transactionTypeData.value = null
   }
 
+  function processTransactionUpdate(message) {
```

Comment:
the `process` prefix it's not really clear to me what it does or what it returns. Would it make sense to name it `updateTransactionTypeData`, or similar?

Reply:
I named it this way to preserve consistency with the other process function but sure, I don't have any strong preferences here. Updated accordingly with your recommendation